lua/state_lua.cpp

```diff
@@ -164,8 +163,6 @@ int pushMember(
     (*f)->incref();
     luaL_getmetatable(L, "torchcraft.Frame");
     lua_setmetatable(L, -2);
-  } else if (m == "frame_string") {
-    lua_pushlstring(L, s->frame_string.c_str(), s->frame_string.length());
```

Comment:
This used to hold the text-serialzed frame. It wasn't used and wasn't useful; confirmed with @ebetica that we can junk it.